kubevirtci

```diff
@@ -168,9 +168,11 @@ function kubevirtci::create_cluster() {
 	export IMAGE_REPO=k8s.gcr.io
 	export CRI_PATH="/var/run/containerd/containerd.sock"
 	template=templates/cluster-template.yaml
-
 	if [ ! -f $template ]; then
-		template="https://github.com/kubernetes-sigs/cluster-api-provider-kubevirt/blob/main/templates/cluster-template.yaml"
+		#can't pass the url directly because clusterctl doesn't resolve the redirects properly.
+		mkdir -p templates
+	    curl -L "https://github.com/kubernetes-sigs/cluster-api-provider-kubevirt/releases/download/$CAPK_RELEASE_VERSION/cluster-template.yaml" --output $template
```

Comment:
formatting looks odd here.

Reply:
I don't understand what you mean, it looks normal to me.